### ISSUE TITLE
Flutter Sprint 

### ISSUE THREAD

**Issue by akshayrb22** [so-sc/open-design#12]
## Event Description:
https://github.com/so-sc/war-room/issues/15
## Design Requirements:
Light and simple design with the Flutter logo. Again, it should gel well with the SOSC website design guidelines.
#### Completion Date:
Check all that is applicaple, add new ones if needed, remove the lines which aren't required.
- [ ] Poster (print)
- [ ] Poster (Social Media)
- [ ] SOSC Website Event Cover

Any further details @ekokratos , @melwinlobo18 , @ShreyasBaliga ?

**Comment by Dhanush-Bangera** [comment]
@so-sc/designteam Who's ready to take this up?

**Comment by dhanushp** [comment]
Not stale :)

**Comment by akshayrb22** [comment]
Not stale. :+1:

**Comment by akshayrb22** [comment]
Let's keep this issue open till the third workshop is complete. As it's a series of workshops, maybe the theme of it can be the same. Or maybe we should leave it just the way it is and change it to Flutter 201? What do you think @dhanushp ?

**Comment by dhanushp** [comment]
Yeah let this issue be open. We will go with the same design and make some changes to current design and make it Flutter 201. Let's get it more attractive in our Instagram feed :smiley:
